Makefile

```diff
@@ -195,18 +197,27 @@ endif
 ## --------------------------------------
 
 .PHONY: local-setup
-local-setup: docker-build ## setup and run sync controller locally
+local-setup: docker-build local-setup-e2e ## setup and run sync controller locally
+
+	kubectl apply -f ./hack/localsetup/e2e-providerspc.yaml
+	kubectl apply -f ./hack/localsetup/e2e-secret-sync.yaml
+
+.PHONY: local-setup-e2e
```

Comment:
Divide this target into `setup-kind-cluster` and `helm-manifest-install`.

For local setup it should look like
```shell
local-setup: docker-build setup-kind-cluster helm-manifest-install
```
Create another target called `e2e-setup`. It should look like:
```shell
e2e-setup: $(BATS) $(ENVSUBST) 
```
We can keep adding binaries here that we might need in future for e2e setup.

And, for e2e test it should look like
```shell
run-e2e-provider-tests: e2e-setup docker-build setup-kind-cluster helm-manifest-install
```